### ISSUE TITLE
AndroidX compatibility : Migrating to AndroidX

### ISSUE THREAD

**Issue by faisalramdan17** [CaiJingLong/flutter_photo_manager#42]
As we are not sure when this will be merged, you can adjust your `pubspec.yaml` in the meantime:

```
flutter_full_pdf_viewer:
    git:
      url: https://github.com/faisalramdan17/flutter_full_pdf_viewer.git
      ref: android-x
```

_Originally posted by @faisalramdan17 in https://github.com/CaiJingLong/flutter_photo_manager/pull/41#issuecomment-465924587_

**Comment by CaiJingLong** [comment]
Are you sure this issue submission location is correct?

**Comment by faisalramdan17** [comment]
Yes, I have tested in my code..
It was working..

**Comment by CaiJingLong** [comment]
Merged your PR #41 , You will see changes in the next version.
